airflow/models/serialized_dag.py

```diff
@@ -65,6 +66,11 @@ class SerializedDagModel(Base):
     fileloc_hash = Column(BigInteger, nullable=False)
     data = Column(sqlalchemy_jsonfield.JSONField(json=json), nullable=False)
     last_updated = Column(UtcDateTime, nullable=False)
+    # TODO: Make dag_hash not nullable in Airflow 1.10.13 or Airflow 2.0??
```

Comment:
We can make it not nullable and add default value in the migration ("Hash not calculated yet"). This way it will be refreshed with the next run automagically and we will not worry about nullability.

Reply:
Good point. I was tempted to do that, the only reason I didn't do it was to avoid any impression that it is a hash but I think that might not be a problem.